legion/engine/physics/physx/systems/physx_physics_system.hpp

```diff
@@ -12,18 +17,39 @@ namespace legion::physics
     {
     public:
 
+        PhysXPhysicsSystem();
```

Comment:
I can't find the definition of this constructor? also i don't think it should be necessary to have any defined constructors or destructors?

Reply:
its on the cpp file on line 18, it adds the instance counter. But I guess this can be done on setup instead